CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ These functions and structs can be referenced in both code and blueprints it may
 - You can now disable logging to spatial for local and/or cloud deployments from the GUI (Project Settings -> Runtime Settings -> Logging). The command line argument -NoLogToSpatial can still be used for that as well.
 - Servers now log a warning message when detecting a client has timed out.
 - Handover is now optional depending on whether the load balancing strategy implementations require it . See `RequiresHandoverData`
+- Improved the failed hierarchy migration logs. The logs are now more specific, the frequency of repeated logs is suppressed and cross-server migration diagnostic logs have been added.
```

Comment:
This message looks to invalidate the changelog below - worth condensing both

Reply:
Yes, thanks for spotting that. I originally condensed them into one but I think a merge put the old one back again.